Sources/NIOTransportServices/NIOTSConnectionBootstrap.swift

```diff
@@ -154,6 +154,12 @@ public final class NIOTSConnectionBootstrap {
         return self
     }
 
+    /// Specifies a type of Multipath service to use for this connection, instead of the default 
+    /// service type for the event loop.
+    public func withMultipath(_ type: NWParameters.MultipathServiceType) -> Self {
```

Comment:
We should probably add this to the listener bootstrap too.

Reply:
I've added this function to listeners too. Note however that [MPTCP doesn't seem to be supported](https://www.mptcp.dev/macOS.html) on the server side for the moment (as it generates warnings when trying to connect to this socket). Perhaps adding a warning about it somewhere would help ?

Reply:
Yeah, that's a good idea. Let's add a warning to the doc comment on the listener bootstrap.

Reply:
Done in the latest commit ! Thank you for your feedback !